task_1/scss/general.scss

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  overflow-x: hidden;
+  scroll-behavior: smooth;
+}
+
+.wrapp {
+  max-width: 1440px;
```

Comment:
![image](https://user-images.githubusercontent.com/17281920/120363095-6cf9bb00-c314-11eb-96d6-87b476e22354.png)

as I see max-width 1200px (1170 + 15px padding on left and right)